src/components/PrescriptionForm.tsx

```diff
@@ -191,19 +230,104 @@ const PrescriptionForm: FunctionComponent<PrescriptionFormProps> = ({
+                      name='lastname'
+                      type='text'
+                      value={patientState.lastname}
+                      onChange={handleChangeNewPatient}
+                    />
+                  </Form.Group>
+
+                  <Form.Group as={Col} className='mt-2'>
+                    <Form.Label><FontAwesomeIcon icon={['fas', 'id-card']} /> {t('form.firstName')}</Form.Label>
+                    <Form.Control
+                      name='firstname'
+                      type='text'
+                      value={patientState.firstname}
+                      onChange={handleChangeNewPatient}
+                    />
+                  </Form.Group>
+
+                  <Form.Group as={Col} className='mt-2'>
+                    <Form.Label><FontAwesomeIcon icon={['fas', 'address-card']} /> {t('form.emailAddress')}</Form.Label>
+                    <Form.Control
+                      name='address'
+                      type='text'
+                      value={patientState.address}
+                      onChange={handleChangeNewPatient}
+                    />
+                  </Form.Group>
+
+                  <Form.Group as={Col} className='mt-2'>
+                    <Form.Label><FontAwesomeIcon icon={['fas', 'mobile']} /> {t('form.phone')}</Form.Label>
+                    <Form.Control
+                      name='phone'
+                      type='text'
+                      value={patientState.phone}
+                      onChange={handleChangeNewPatient}
+                    />
+                  </Form.Group>
+
+                  <Form.Group as={Col} className='mt-2'>
+                    <Form.Label><FontAwesomeIcon icon={['fas', 'city']} /> {t('form.city')}</Form.Label>
+                    <Form.Control
+                      name='city'
+                      type='text'
+                      value={patientState.city}
+                      onChange={handleChangeNewPatient}
+                    />
+                  </Form.Group>
+
+                  <Form.Group as={Col} className='mt-2'>
+                    <Form.Label><FontAwesomeIcon icon={['fas', 'city']} />  {t('form.zipPostal')}</Form.Label>
+                    <Form.Control
+                      name='zip_code'
+                      type='text'
+                      value={patientState.zip_code}
+                      onChange={handleChangeNewPatient}
+                    />
+                  </Form.Group>
+                  <div className='d-flex align-items-center mt-4'>
+                    <Button variant='success' type='button' onClick={handleNewPatientSubmit}>
+                      {t('navigation.validate')}
+                    </Button>
+                    <Button className='btn btn-primary ms-4' href='/patients'>
+                      {t('navigation.return')}
+                    </Button>
+                  </div>
+                </Form>
+              </Modal.Body>
+            </Modal>
```

Comment:
Should the error appear in this modal instead of the parent component?
![image](https://github.com/mynotif/mynotif-frontend/assets/24973/d3ed20e3-bb04-4e88-a14e-ef7c5c985e01)

Reply:
no error should appear in the children component. I  resolved